docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -96,7 +103,8 @@ you will need to split them up when deploying {rhos_acro} {rhos_curr_ver}. Each
 backend driver needs to use its own instance of the `manila-share`
 service.
 * If a storage backend driver needs a custom container image, find it on the
-https://catalog.redhat.com/software/containers/search?gs&q=manila[RHOSP Ecosystem Catalog]
+https://catalog.redhat.com/software/containers/search?gs&q=manila[RHOSP Ecosystem Catalog] 
+//kgilliga: Should this link to the RH Ecosystem Catalog appear downstream only?
```

Comment:
Perhaps but i'm less concerned about linking downstream from upstream docs than vice versa. So personally i wouldn't mind keeping this.